doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Power manager keep alive module for buttons
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |buttons_pm_keep_alive| of the :ref:`lib_caf` (CAF) is responsible for generating :c:struct:`keep_alive_event` events on any configured button press.
+
+Configuration
+*************
+
+To use the module, you must enable the following Kconfig options:
+
+* :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE` - This option enables the |buttons_pm_keep_alive|.
```

Comment:
Maybe we should inform that this option is selected by default when buttons and keep_alive_events are enabled ? So that user will be aware about it.

Reply:
Lets agree if we should enable it by default before any change in documentation.

Reply:
Maybe we could state that this module is enabled by default if both buttons and power manager are enabled? No additional steps are needed to enable the module with current configuration.

Reply:
ok.